share/p2p/shrexnd/params.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/celestiaorg/celestia-node/share/p2p"
 )
 
-const protocolString = "/shrex/nd/0.0.1"
+const protocolString = "/shrex/nd/0.0.2"
```

Comment:
I had a question about this while I was writing docs for peer manager and exploring getters, why is it that versions in other protocols we have are prefixed with `v` and `shrex/nd` is not. (e.g: `/shrex/nd/v0.0.2`

Maybe since this is a breaking PR we can fix this and make this protocol uniform with the rest.

Reply:
you still didnt fix

Reply:
it is fixed, just not showing in the diff